cylc/uiserver/resolvers.py

```diff
@@ -190,6 +190,8 @@ async def mutator(self, info, *m_args):
         w_ids = [
             flow[WORKFLOW].id
             for flow in await self.get_workflows_data(w_args)]
+        if not w_ids:
+            return [{'error': 'No matching workflows'}]
```

Comment:
I think it should be:

```suggestion
            return [{
                'errors': [{'message': 'No matching workflows'}]
            }]
```

Reply:
It won't result in the format we want.. We'd have to raise or return some sort of `ExecutionResult` class or something